From: Jiri Slaby <jslaby@suse.cz>
Date: Thu, 29 Sep 2016 12:03:37 +0200
Subject: Revert "Input: i8042 - set up shared ps2_cmd_mutex for AUX ports"
Patch-mainline: never, kabi

This reverts commit 1bc74ff3eeb249ab98674bb31e92abee566bb3d5, upstream
commit 47af45d684b5f3ae000ad448db02ce4f13f73273. It is a fix for
e5d9fc669fe391ba7bd187873051fec42d7fc7df, upstream commit
4097461897df91041382ff6fcd2bfa7ee6b2448c, but we are going to revert
it too.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/input/serio/i8042.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/input/serio/i8042.c b/drivers/input/serio/i8042.c
index 09da61eb07fa..7ecca05bd7a5 100644
--- a/drivers/input/serio/i8042.c
+++ b/drivers/input/serio/i8042.c
@@ -1258,7 +1258,6 @@ static int __init i8042_create_aux_port(int idx)
 	serio->write		= i8042_aux_write;
 	serio->start		= i8042_start;
 	serio->stop		= i8042_stop;
-	serio->ps2_cmd_mutex	= &i8042_mutex;
 	serio->port_data	= port;
 	serio->dev.parent	= &i8042_platform_device->dev;
 	if (idx < 0) {
-- 
2.10.0

